File: Cloth-Store-Agency-BE/ct-core-model/src/main/java/com/example/ctcoremodel/SupplierModel.java
package com.example.ctcoremodel;

import com.example.ctcommon.enums.StatusSupplier;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@AllArgsConstructor
@Data
@NoArgsConstructor
public class SupplierModel {
    private String id;
    private String code;
    private String name;
    private String phone;
    private String address;
    @Enumerated(EnumType.STRING)
    private StatusSupplier status;
    private LocalDateTime dateCreate;
    private LocalDateTime dateUpdate;
}

File: Cloth-Store-Admin-FE/src/app/core/apis/Dtos/Supplier.model.ts
import {DatePipe} from "@angular/common";


export class SupplierModel{
  id! : string | null;
  code! : string | null;
  name! : string | null;
  phone! : string | null;
  address! : string | null;
  status!: string | null;
  dateCreate! : Date | null;
  dateUpdate! : Date | null;
}

